iOS-School-02_stack.playground/Contents.swift

```diff
@@ -0,0 +1,101 @@
+// MARK: - Stack
+/**
+ Provides a standard interface for operations on a collection,
+ which is organized according to the principle last in first out, called a `stack`.
+ Using class methods, it is possible to add an element to the collection,
+ remove an element from the collection and read the last element of the collection.
+ */
+class Stack {
+    /// An array with the elements of the stack collection.
```

Comment:
Be simple:
```suggestion
    /// An array with the elements of the stack.
```

Reply:
nit:

also I know this is an array, you can simplify it as:
```suggestion
    /// Contains elements of the stack.
```

Reply:
Corrected to "сontains elements of the stack".